nix/modules/home-manager.nix

```diff
@@ -144,6 +144,18 @@ in {
         description = "Show search results immediately when Anyrun starts";
       };
 
+      persistState = mkOption {
```

Comment:
@NotAShelf should have a look at the Nix parts to see if they are fine.

Reply:
Nix side looks fine, nothing to note.